NodeFirefox/Dockerfile

```diff
@@ -42,3 +42,6 @@ COPY generate_config /opt/bin/generate_config
 # To run a command as administrator (user "root"), use "sudo <command>". See "man sudo_root" for details.
 # When logging into the container
 RUN sudo echo ""
+
+# Generating config inside the image rather than with entry_point.sh
```

Comment:
Why do you need to change the Dockerfile? Besides, please read the note on top and only change the .txt file.